public/forms/builder/form.ttl

```diff
@@ -11,96 +11,96 @@
 # form
 ##########################################################
 ext:form a form:Form, form:TopLevelForm ;
-  form:includes ext:propertyGroupL;
+  form:includes ext:sectionL;
   form:includes ext:formNodesL;
   form:includes ext:formListingL;
   form:includes ext:formListingTableL.
 
 ##########################################################
-#  property-group
+#  Section
 ##########################################################
-ext:mainPg a form:PropertyGroup;
+ext:mainPg a form:Section;
     sh:name "Form Builder" ;
     sh:order 0 .
 
-ext:propertyGroupPg a form:PropertyGroup;
+ext:sectionPg a form:Section;
```

Comment:
`ext:SectionPg` is not correct anymore as `Pg` stands for `PropertyGroup` The same for the subjects below.